index.html

```diff
@@ -0,0 +1,250 @@
+</div>
+</section>
+<!-- Section 5 closed -->
+
+<!-- Section 6 started -->
+<!-- Row five started-->
+<section class="row5">
+  <div class="r3left">
+    <h2 class="tonic">Uber Navigation</h2>
+    <ul>
+      <li class="canopli">
+         <a href="/someurl" class="sec3rlink31">Uber</a>
+          <b class="sec3p">.</b>
+          <a href="/someurl" class="sec3rlink31">Lead Developer</a>
+           <b class="sec3p">.</b>
+         <a href="/someurl" class="sec3rlink31">2018</a>
+       </li>
+    </ul>
+<p class="paragraph1">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+
+<ul class="ad">
+<li class="canopli2">
+     <a href="/someurl" class="sec3rlink32">HTML</a>
+     <a href="/someurl" class="sec3rlink32">Ruby on rails</a>
+     <a href="/someurl" class="sec3rlink32">CSS</a>
+     <a href="/someurl" class="sec3rlink32">JAVASCRIPT</a>
+ </li>
+</ul>
+<button type="submit" class="bb sec3rbutton sec3rlink4"><span class="d">See Projects</span></button>
+  </div>
+  <div class="right">        
+    <img src="images/d.png" alt="">
+  </div>
+</section>
+
+<!-- Sectoin 6 closed -->
+
+<!-- Section 7 started -->
+<section class="row6">
+  <div class="r3left">
+    <h2 class="tonic ">I’m Aziz Glad to see you!</h2>
+    <h4  class="paragraph1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
+<h5 class="ets1" >ETS CONNECT</h5>
+<ul class="social-menue" >
+<li><img src="images/twitter.PNG" alt="abc"  class="lsls"></li>
+<li><img src="images/linkedinicon.PNG" alt="abc" class="lsls"></li>
+<li> <img src="images/medium.PNG" alt="abc" class="lsls"></li>
+<li> <img src="images/github.PNG" alt="abc" class="lsls"></li>
+<li><img src="images/angellist.PNG" alt="abc" class="lsls"></li>
+</ul>
+
+<button type="submit" class="bb sec3rlink6"><span class="d">See Projects</span></button>
+  </div>
+  
+
+  <div class="right">        
+    <h2 class="tonic sec3rlink31">Languages</h2>
+    <ul>
+      <li class="canopli6">
+         <a href="/someurl" class="sec3rlink316"><img src="images/jas.png" alt=""></a>
+         <a href="/someurl" class="sec3rlink316"><img src="images/css1.png" alt=""></a>
+          <a href="/someurl" class="sec3rlink316"><img src="images/n.png" alt=""></a>
+        
+       </li>
+    </ul>
+
+<ul class="ad">
+<li class="canopli2">
+  
+     <a href="/someurl" class="sec3rlink32">JAVASCRIPT</a>
+     <a href="/someurl" class="sec3rlink32">HTML</a>
+     <a href="/someurl" class="sec3rlink32 r6326">CSS</a>
+ </li>
+</ul>
+
+<ul>
+<li class="canopli6">
+ <a href="/someurl" class="sec3rlink316 ks"><img src="images/frame.PNG" alt=""></a>
+ <a href="/someurl" class="sec3rlink316"><img src="images/ks.png" alt=""></a>
+
+</li>
+</ul>
+
+  </div>
+</section>
```

Comment:
Kindly make this section resemble Figma design as closely as possible for both mobile and desktop 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/187714322-cd9fc88d-7801-4afb-a36c-ba61b65839ab.png)
![image](https://user-images.githubusercontent.com/73714615/187714474-3db6f260-99ba-4583-86aa-1d3dbd404729.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187714551-39cf4351-2891-4a25-b712-23fe8c167ce2.png)
![image](https://user-images.githubusercontent.com/73714615/187714589-9e016757-c26b-4a5d-9c7e-514a71868258.png)

Reply:
Kindly, I recommend you to follow the suggestions of the previous reviewer, this will help your project to look more PRO 😉